Libraries/LibGfx/Bitmap.cpp

```diff
@@ -40,11 +40,13 @@ IntSize oriented_size(IntSize size, Gfx::ExifOrientation orientation)
     VERIFY_NOT_REACHED();
 }
 
-int Bitmap::width_considering_exif() const {
+int Bitmap::width_considering_exif() const
```

Comment:
Please move those changes to the first commit, so each commit is correct by itself. [This video](https://www.youtube.com/watch?v=ElRzTuYln0M) explains how to do that pretty well if you're not familiar with it.

If you want, there's a [git commit hook](https://github.com/LadybirdBrowser/ladybird/blob/master/CONTRIBUTING.md#commit-hooks) available, which runs clang-format automatically each time you try to commit, and makes sure your commit messages match our rules too. (It's quite handy. 😅)